### DISCUSSION TITLE
Double calls in React

### DISCUSSION THREAD

**Discussion by adamhenderson** [quarkiverse/quarkus-quinoa#111] [General]
Hi @ia3andy (Andy), just watched your Quarkus Insight 92 Quinoa stream and you saw double calls being made, I have seen this also and believe it is related to React Strict mode: https://reactjs.org/docs/strict-mode.html which intentionally double renders components which could be causing this.

**Comment by ia3andy** [comment]
Thanks Adam, yeah that might be it. 

IMO the best solution is still to have a timestamp in the state (for caching) to avoid reloading because of the component life.

**Reply by ia3andy** [reply]
@adamhenderson  Did you learn anything? Because you already knew Quinoa by heart 😅
